### PR TITLE
Added always after relationship algorithm

### PR DESCRIPTION
**Pull request by AveraGE0**

### CODE REVIEW

File: src/components/logic/always_after.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            return False
+
+        activity_projection = self.project_trace(trace, [activity1, activity2])
+
+        if activity1 not in activity_projection:
+            return True
+        elif activity_projection[-1] == activity2:
+            return True
+        else:
+            return False
```
Comment (Bohne9) [inline src/components/logic/always_after.py:28]: Can you change that bit of code to:

`return activity1 not in activity_projection or activity_projection[-1] == activity2`

I think this is much cleaner. 😄

File: src/components/logic/always_after.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from src.components.logic.relationship import Relationship
+from src.components.util.xes_importer import XES_Importer
+from pm4py.algo.discovery.log_skeleton import algorithm as lsk_discovery
+
+
+class AlwaysAfter(Relationship):
+    """ Implementation of the always after relationship"""
+
+    def __init__(self, traces):
+        super().__init__(traces)
```
Comment (Bohne9) [inline src/components/logic/always_after.py:12]: Is the Linter not asking for a docstring here?

Comment (Bohne9) [reply src/components/logic/always_after.py:12]: Apparently not, otherwise the test would not pass.

**Review by Bohne9 — APPROVED** [review]

File: src/components/logic/always_after.py (hunk 2 above)
Comment (AveraGE0) [reply src/components/logic/always_after.py:12]: I was about to change it haha, also pycharm usually shows all Linter warnings, but in this case it didnt show anything

**Pull request merged**